src/languages/wescheme/WeschemeParser.d.ts

```diff
@@ -0,0 +1,8 @@
+import { Language } from "../../CodeMirrorBlocks";
+
+interface WeschemeParser {
```

Comment:
Will other block languages have to declare this as well?

Reply:
nope.

Reply:
All the typescript gets removed/converted to javascript for the actual npm package. So it is completely optional for code that depends on the npm package to use typescript.